ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml

```diff
@@ -220,7 +220,6 @@ tests:
     workflow: openshift-e2e-aws
 - as: e2e-agent-compact-ipv4
   cluster: build05
-  optional: true
```

Comment:
I'd rather leave them optional, but lets make them always_run? Installer team may introduce a breaking change and should not depend on OKD bugs to land it.

Another option is to add agent-compact jobs to OKD nightlies testing to make sure we catch regressions in the entire payload

Reply:
Putting `always_run: true` anyhow will consume an ofcir instance for all the PRs, even those ones that are not related at all for the agent. It could have more sense to have it always executed in an informing (periodic) job maybe, and leave the presubmits as they are now (not blocking)

Reply:
Right, informing job sounds like a better idea